webosose/ares-cli
# Support capture screen

:Release Notes:
Support capture in ares-device

:Detailed Notes:
Implement capture screen feature to developer watch target display
- Add ares-device --capture-screen(-c)
- Add capture option description in ares-device help  
- Update positive/negative TC for capture option 

:Testing Performed:
1. Pass unit test on ose and auto
2. Pass eslint
3. Verify with CLI commands

- Check capture file generated with _targetname__display0__time_.png
$ ./ares-device.js -c 
Create rpi4_display0_20210115095912.png to /home/ye0607kim/project/github/ares-cli/bin
Success

- Give display option and capture file name should be _targetname__display1__time_.png
$./ares-device.js -c -dp 1
Create rpi4_display1_20210115095826.png to /home/ye0607kim/project/github/ares-cli/bin
Success
                                      
- Give directory path with display option
$ ./ares-device.js -c webOSCap -dp 1
Create rpi4_display1_20210115095944.png to /home/ye0607kim/project/github/ares-cli/bin/webOSCap
Success

- Give file name and format(png, bmp, jpg) and check the generated file name & format
$ ./ares-device.js -c webOSCap/test1.bmp _[or test1.jpg or test1.png]_
Create test1.bmp  _[or test1.jpg or test1.png]_ to /home/ye0607kim/project/github/ares-cli/bin/webOSCap
Success

- Check /tmp/aresCapture has been removed from device target after command working completed. 

:Issues Addressed:
[PLAT-125564] Implement ares-device --capture-screen
